common/include/systemctrl.h

```diff
@@ -36,7 +38,7 @@ extern "C"{
 //#define sctrlKernelQuerySystemCall sceKernelQuerySystemCall
 
 // Prologue Module Start Handler
-typedef void (* STMOD_HANDLER)(SceModule2 *);
+typedef int (* STMOD_HANDLER)(SceModule2 *);
```

Comment:
The entire CFW is written with this function as a void, changing it to int could lead to unsidered results.

Reply:
Okay I will change that thanks.